docfx_project/docs/intro.md

```diff
@@ -0,0 +1,88 @@
+# Introduction
```

Comment:
A lot of this document contains the same information already in https://github.com/natemcmaster/CommandLineUtils/blob/master/docfx_project/index.md. Can you merge these two files?

Reply:
Yep, my idea was to make the `index.md` file more of a landing page to link directly to some of the core concepts. For example, same as on the landing page for the ASP.NET Core docs:
https://docs.microsoft.com/en-us/aspnet/

I also did something similar for the OpenIddict docs:
https://openiddict.github.io/openiddict-documentation/

Its just there is not much to link to, yet ;) So I can perhaps just give a brief intro paragraph and link to the docs and samples. Would that be ok for now?

Reply:
I like what you did with OpenIddict. I think it would be good to have a similar landing page. So, in this change, are you going to remove the duplicated content between index.md and docs/intro.md?

Reply:
Yep, that was the idea. As soon as we get more content ;)